packages/contentstack-export/package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "@contentstack/cli-cm-export",
   "description": "Contentstack CLI plugin to export content from stack",
-  "version": "1.3.1",
+  "version": "1.3.2",
```

Comment:
@abhinav-from-contentstack This version is already bumped

Reply:
@netrajpatel ok, I'll fix this right away

Reply:
done